src/libraries/pkg/test/testPackages.proj

```diff
@@ -13,7 +13,9 @@
     <!-- support override via commandline -->
     <RuntimesToInclude Condition="'$(RuntimesToInclude)' != ''" Include="$(RuntimesToInclude)" />
     <TargetFrameworksToInclude Condition="'$(TargetFrameworksToInclude)' != ''" Include="$(TargetFrameworksToInclude)" />
-
+    <TestPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.nupkg" />
```

Comment:
Make sure you exclude `ExcludePackages` and honor `PackagesToTest`

Reply:
done and tested locally that both scenarios are working